src/3rd_party/CMakeLists.txt

```diff
@@ -9,10 +9,20 @@ add_subdirectory(./faiss)
 include_directories(./faiss)
 
 if(COMPILE_CPU)
-  if(NOT GENERATE_MARIAN_INSTALL_TARGETS)
+  if(NOT GENERATE_MARIAN_INSTALL_TARGETS AND NOT ARM)
     set(INTGEMM_DONT_BUILD_TESTS ON CACHE BOOL "Disable intgemm tests")
     add_subdirectory(./intgemm)
   endif()
+
+  if(USE_RUY_SGEMM)
```

Comment:
Is this worth a short comment in the code why it's disabled?

Reply:
it is because the default codepath does not use ruy so we don't want to add the subdirectories to the build unless it is explicitly enabled. I can add a comment

Reply:
Adding a comment will be useful.

Reply:
added comment